### PR TITLE
Add avocado ci-fix

### PR DESCRIPTION
**Pull request by ruowan**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: documentation/ci-fix.md [hunk 1]
```diff
@@ -44,3 +44,22 @@ oav validate-spec <swagger-spec-path>
...
 Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/static-validation/static/errors/default)
 Refer to [Semantic and Model Violations Reference](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) for detailed description of validations and how-to-fix guidance.
+
+## Avocado
+
+Run avocado locally:
+
+```
+npm install -g @azure/avocado
+
```
Comment (akning-ms) [inline documentation/ci-fix.md:54]: Should we add step here,?e.g. cd <your RP directory>

Comment (ruowan) [reply documentation/ci-fix.md:54]: No. `npm install -g` mean install in root node_module. You can use avocado in any directory.

File: documentation/ci-fix.md [hunk 2]
```diff
@@ -44,3 +44,22 @@ oav validate-spec <swagger-spec-path>
...
+avocado
+```
+
+When type avocado in command line, avocado will validate in the current directory.
+
+NOTE: When running in azure devops Avocado only report new errors involved in PR, but ignore the previous existing errors. So when running in local machine, Avocado report all errors.
```
Comment (josefree) [inline documentation/ci-fix.md:60]: The NOTE is a little bit confusing. Some tuning based my understanding...

Note: when running in Azure Devops (what do you referring to? Do you mean PR pipeline? ), Avocado only reports errors with file updates in the PR, but ignore errors existing in the base. However, when running Avacado against local directory, it reports all errors existing in the files.

Comment (ruowan) [reply documentation/ci-fix.md:60]: Yes. It's Azure pipeline. updated the note.

File: documentation/ci-fix.md (hunk 1 above)
Comment (akning-ms) [reply documentation/ci-fix.md:54]: I mean cd to the RP folder before run avocado

Comment (ruowan) [reply documentation/ci-fix.md:54]: Currently, Avocado can't support running in RP folder.  Avocado running process based on root folder.  I can design a feature to support it.

**Review by akning-ms — APPROVED** [review]

**Pull request merged**